css/layout.css

```diff
@@ -29,6 +29,16 @@
   justify-content: center;
 }
 
+.modal-content {
+  position: relative;
+  background-color: #fefefe;
+  margin: 25% auto;
+  padding: 20px;
+  width: 90%;
+  height: 80%;
```

Comment:
What happens if you remove this height? You shouldn't set a height on an element unless it is _absolutely_ necessary.